oci/spec_opts.go

```diff
@@ -701,11 +701,8 @@ func WithUIDGID(uid, gid uint32) SpecOpts {
 func WithUserID(uid uint32) SpecOpts {
 	return func(ctx context.Context, client Client, c *containers.Container, s *Spec) (err error) {
 		setProcess(s)
-		if c.Snapshotter == "" && c.SnapshotKey == "" {
-			if !isRootfsAbs(s.Root.Path) {
-				return errors.New("rootfs absolute path is required")
-			}
-			user, err := UserFromPath(s.Root.Path, func(u user.User) bool {
+		f := func(root string) error {
```

Comment:
Maybe the function can be named to `setUser`, similar to how its named [here](https://github.com/containerd/containerd/blob/main/oci/spec_opts.go#L817)  for GID

Reply:
Thanks, I renamed it.